modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -22,6 +22,20 @@
 #include "backends/ie/util.hpp"
 #include "backends/ie/giebackend/giewrapper.hpp"
 
+#ifdef HAVE_NGRAPH
```

Comment:
Should tests link with `ngraph::ngraph` too? I don't remember if it is the case for CMake.

Reply:
That's a good point, but `ngraph` is enabled by using `-DWITH_NGRAPH=ON` and this flag isn't mandatory to use `IE` itself.
So if user just do `cmake` as same as usual: `-DWITH_INF_ENGINE=ON` ngraph won't be found.


@ilya-lavrenov Could you help, please? Do you know whether `ngraph` will be mandatory part of `IE` in upcoming `OpenVINO 2.0` release?